migen/build/platforms/quickfeather.py

```diff
@@ -0,0 +1,96 @@
+# This file is Copyright (c) 2020 Antmicro <www.antmicro.com>
+
+from migen.build.generic_platform import *
+from migen.build.quicklogic import QuicklogicPlatform, JLinkProgrammer, OpenOCD
+
+_io = [
+        ("rgb_led", 0,
+            Subsignal("r", Pins("22")),
+            Subsignal("g", Pins("21")),
+            Subsignal("b", Pins("18")),
+            IOStandard("LVCMOS33"),
+        ),
+        ("i2c", 0,
+            Subsignal("scl", Pins("0")),
+            Subsignal("sda", Pins("1")),
+            IOStandard("LVCMOS33"),
+        ),
+        ("accel", 0,
+            Subsignal("int", Pins("3")),
+            IOStandard("LVCMOS33"),
+        ),
+        ("user_btn", 0, Pins("6"), IOStandard("LVCMOS33")),
+        ("serial_debug", 0,
+            Subsignal("clk", Pins("14")),
+            Subsignal("data", Pins("15")),
+            IOStandard("LVCMOS33"),
+        ),
+        ("spi", 0,
+            Subsignal("clk", Pins("34")),
+            Subsignal("mosi", Pins("38")),
+            Subsignal("miso", Pins("36")),
+            IOStandard("LVCMOS33"),
+        ),
+        ("serial", 0,
+            Subsignal("tx", Pins("44")),
+            Subsignal("rx", Pins("45")),
+            IOStandard("LVCMOS33"),
+        ),
+        ("IO_CLK",    0, Pins("63"), IOStandard("LVCMOS33")),
+        ("IO_CLK",    1, Pins("61"), IOStandard("LVCMOS33")),
+        ("IO_CLK",    2, Pins("33"), IOStandard("LVCMOS33")),
+        ("IO_CLK",    3, Pins("32"), IOStandard("LVCMOS33")),
+        ("IO",    0, Pins("4"), IOStandard("LVCMOS33")),
```

Comment:
Not too familiar with this board, but wouldn't it make sense to define those as a connector?

Reply:
I declared 3 connectors according to the QuickFeather schematic ([link](https://github.com/m-labs/migen/pull/222/files#diff-8414900effb02b7160baa94cb0d7b7b3063d78f18a6a13ebfef48c846eb202a7R41-R45))